### PR TITLE
Fixed the comment alignment

### PR DESCRIPTION
**Pull request by mohammedali101**
- Fixed alignment for the comments on the post page.
- The user ID, creation date, and comment body are all on the left side. 
- Please refer to images below as reference.

left alignment

example 2

### CODE REVIEW

**Comment by mohammedali101** [comment]
- 1st picture: message when user tries signing up with existing email in the signup page
- 2nd picture: the comment section modified with edited padding/divider to have line displayed above body/user icon to show who added the comment

validation

comment UI

**Comment by tanvirahmed725** [comment]
The error message should be more meaningful when the user attempts to create an account with an existing email or username. I.e 'An account already exists with that email or username.'

For the comment UI have the user name on the same line as the user icon.

**Comment by mohammedali101** [comment]
Updated comment page UI with user icon similar to how it is for posts and comment body is also aligned to the left:
Screen Shot 2022-11-01 at 8 00 38 PM

File: lib/util/cognito/auth_util.dart [hunk 1]
```diff
@@ -67,21 +68,32 @@ class AuthUtility {
...
+    } on CognitoUser catch (e) {
+      // throw exception when user enters existing email
+      print("An account with this email already exists:  $e");
+      return {
+        'status': false,
+        'message': 'Please sign in with the email associated with this account.'
+      };
     } catch (e) {
-      print("An error occurred while creating the user's account:  $e");
+      print("An error occurred while authenticating user:  $e");
```
Comment (abe-mused) [inline lib/util/cognito/auth_util.dart:89]: > An error occurred while authenticating user

This is the `signup` method?

**Pull request closed without merging**